common/src/format.rs

```diff
@@ -338,7 +346,10 @@ impl FormatSpec {
         if pad_cnt > 0 {
             // separate with 0 padding
             let sep_cnt = disp_digit_cnt / (inter + 1);
-            let padding = "0".repeat((pad_cnt - sep_cnt) as usize);
+            let padding = match use_whitespace_padding {
+                true => " ".repeat((pad_cnt - sep_cnt) as usize),
+                false => "0".repeat((pad_cnt - sep_cnt) as usize),
+            };
```

Comment:
`match`ing on a bool is less readable than using `if`-statements

```suggestion
            let padding = if use_whitespace_padding {
                " ".repeat((pad_cnt - sep_cnt) as usize)
            } else {
                "0".repeat((pad_cnt - sep_cnt) as usize)
            };
```

Reply:
```suggestion
            let padding_char = if use_whitespace_padding {
                ' '
            } else {
                '0'
            };
            let padding = padding_char.repeat((pad_cnt - sep_cnt) as usize);
```

Reply:
> 

I found a similar code implemented before(`FormatSpec::compute_fill_string`) and changed it to that instead. Thank you.